### ISSUE TITLE
Transactions have pending status more than 3 days....

### ISSUE THREAD

**Issue by 0x64696d61** [iron-fish/ironfish#1969]
### What happened?

Transactions have pending status more than 3 days.... I can't sent IRON....

ironfish accounts:balance
The balance is: $IRON 0.23818190 ($ORE 23,818,190)
Amount available to spend: $IRON **0.00000000** ($ORE 0)

ironfish accounts:transactions
 pending x       8ba0301d521465ccf6f859cb6be7e1835056557a5d4be0128a5ca319931d5954 x         710        711   1      123704     
 pending x       e5ee10360160578fbee6ce9a3078cc9596c348cc8bf6061b01b34252d794becf x         1011       1012  1      124101     
 pending x       e36a6bd56689027d1afb83cf75fafd4fd927d011608655bba4a02926046fc764 x         1006       1007  1      124306     
 pending x       8440ebf08cfb30f57253075fbd431faf1c592bbf9b6ff1f71df5ee1504cef999 x         1086       1087  1      125552     
 pending x       1793414b03d0837e9b76be488643121b883597a77d6a440269014852b07375e1 x         688        689   1      125598     
 pending x       efa6a8a5fe2159d42c7fe0abe4ac76d9c561a9acce22ba46a7f46caab1f8c290 x         833        834   1      125702     
 pending x       d6ec21d4274e4c7be49757c00dfa9b1ea3f016b5e21ee98d4bdc8ea58e0adaad x         701        702   1      125766     
 pending x       d9b3ac0bdcb56fe6bbc555411829f17cba6e78fb96d7ba975c2089cbce12ab6b x         711        712   1      125841     

### Version

ironfish/0.1.40 linux-x64 node-v16.16.0

### Debug output

```shell
Iron Fish version       0.1.40 @ src
Iron Fish library       0.0.18 @ src
Operating system        Linux x64
CPU model(s)            AMD EPYC 7401P 24-Core Processor
CPU threads             48
RAM total               125.77 GiB
Heap total              4.01 GiB
Node version            v16.16.0
ironfish in PATH        true
Telemetry enabled       true
```

### Graffiti

hardnode

**Comment by 0x64696d61** [comment]
Now balance and transactions are missing :(

`Version              0.1.43 @ src
Node                 STARTED
Node Name            hardnode
Block Graffiti       hardnode
Memory               Heap: 92.90 MiB -> 114.82 MiB / 4.01 GiB (2.3%), RSS: 851.42 MiB (0.7%), Free: 123.12 GiB (2.1%)
P2P Network          CONNECTED - In: 466.22 KB/s, Out: 13.32 KB/s, peers 82
Mining               STARTED - 1 miners, 0 mined
Mem Pool             258 tx
Syncer               SYNCING - 0.36 blocks per seconds, avg time to add block 3241.98 ms
Blockchain           SYNCED @ HEAD 0000000000072a286236273e71db2b045161f8333e62e01928a4f0f74cd5d9aa (133991)
Telemetry            STARTED - 339 <- 1 pending
Workers              STARTED - 500 -> 6 / 6 - 0.07 jobs Δ, 11.42 jobs/s
`

`ironfish accounts:balance
yarn run v1.22.19
$ yarn build && yarn start:js accounts:balance
$ tsc -b
$ cross-env OCLIF_TS_NODE=0 IRONFISH_DEBUG=1 node --expose-gc bin/run accounts:balance
Account - hardnode (bc26fe6)

The balance is: $IRON 0.00000000 ($ORE 0)
Amount available to spend: $IRON 0.00000000 ($ORE 0)`

`ironfish accounts:transactions
yarn run v1.22.19
$ yarn build && yarn start:js accounts:transactions
$ tsc -b
$ cross-env OCLIF_TS_NODE=0 IRONFISH_DEBUG=1 node --expose-gc bin/run accounts:transactions

 hardnode (bc26fe6) - Account transactions

 Status Creator Hash Miner Fee Fee ($ORE) Notes Spends 
 ────── ─────── ──── ───────── ────────── ───── ────── 


Done in 3.29s.`

**Comment by hughy** [comment]
Closing as stale because we've rewritten the wallet since the time that this issue was created.

Please open a new issue if you're continuing to see this issue in new versions.
